Flakey test

The test has been failing here occasionally, even though the output looks successful:
https://github.com/infobloxopen/konk/pull/101/checks?check_run_id=1380932845

```
+ KUBECONFIG=
+ kubectl api-versions --certificate-authority=/nginx/ca.crt --client-certificate=/nginx/tls.crt --client-key=/nginx/tls.key -s https://runner-konk:6443
admissionregistration.k8s.io/v1
admissionregistration.k8s.io/v1beta1
apiextensions.k8s.io/v1
apiextensions.k8s.io/v1beta1
apiregistration.k8s.io/v1
apiregistration.k8s.io/v1beta1
authentication.k8s.io/v1
authentication.k8s.io/v1beta1
authorization.k8s.io/v1
authorization.k8s.io/v1beta1
certificates.k8s.io/v1
certificates.k8s.io/v1beta1
coordination.k8s.io/v1
coordination.k8s.io/v1beta1
discovery.k8s.io/v1beta1
events.k8s.io/v1
events.k8s.io/v1beta1
extensions/v1beta1
node.k8s.io/v1beta1
policy/v1beta1
rbac.authorization.k8s.io/v1
rbac.authorization.k8s.io/v1beta1
scheduling.k8s.io/v1
scheduling.k8s.io/v1beta1
v1

make: *** [test-konk] Error 1
Makefile:58: recipe for target 'test-konk' failed
```
for when test run disappears

--- a/helm-charts/konk/templates/tests/test-connection.yaml
+++ b/helm-charts/konk/templates/tests/test-connection.yaml
@@ -21,12 +21,18 @@ spec:
       do
         sleep 1s
       done
-      kubectl get apiservices
-      KUBECONFIG='' kubectl api-versions \
+      until kubectl get apiservices
+      do
+        sleep 1s
+      done
+      until KUBECONFIG='' kubectl api-versions \
         --certificate-authority=/nginx/ca.crt \
         --client-certificate=/nginx/tls.crt \
         --client-key=/nginx/tls.key \
         -s https://{{ include "konk.fullname" . }}:6443
+      do
+        sleep 1s
+      done
     env:
     - name: KUBECONFIG
       value: /kubeconfig/admin.conf

